localstack/services/awslambda/invocation/lambda_models.py

```diff
@@ -64,6 +64,9 @@
 SNAP_START_SUPPORTED_RUNTIMES = [Runtime.java11, Runtime.java17]
 
 
+BUCKET_ACCOUNT = "949334387222"
```

Comment:
⚠️ This change breaks all existing Lambda pods or persisted lambda functions. This is a breaking change and should be labeled at least as `semver:minor`

Reply:
The recent cross-account enhancements as well, so good to batch them: https://github.com/localstack/localstack/commit/28427b9deedb747fbe0c360a572adaa4c44312a5

Reply:
This is actually a fix for that PR, as it broke persistence for remote fetched layers. Not exactly sure why currently, but it blocks the -ext pipeline.